addons/web/static/src/legacy/action_adapters.js

```diff
@@ -49,9 +49,18 @@ class ActionAdapter extends ComponentAdapter {
                 };
                 core.bus.trigger("DOM_updated");
 
+                this.__widget.el.addEventListener(
+                    "history-back",
+                    this.wowlEnv.config.onHistoryBack
+                );
+
```

Comment:
this part is not necessary I think. Legacy widgets will use trigger_up, and owl components will directly call config.historyBack. Am I missing something?

Reply:
it is apparently but maybe only because of some tests